docs/source/contents/4_draft_mapping.rst

```diff
@@ -1331,6 +1331,297 @@ location
 recordInfo
 ==========
 
+recordIdentifier
+----------------
+
+Use Case
+^^^^^^^^
+
+Unremediated records often contain identifiers for the record. These take a couple of different forms. The Heilman
+collection and Volunteer Voices collections contain this element. In Volunteer Voices the identifier is simply the adminDB
+value with 'record' appended to the beginning (`e.g. volvoices:2352 <https://digital.lib.utk.edu/collections/islandora/object/volvoices%3A2352/datastream/MODS/view>`_).
+
+Justification
+^^^^^^^^^^^^^
+
+As the basic root of the <recordIdentifier> value is already present in the identifier element in all cases and the
+<recordIdentifier> value is never used on its own, there is no reason to retain these values.
+
+Xpath
+^^^^^
+
+"mods:recordInfo" / "mods:recordIdentifier"
```

Comment:
Normally, we express an xpath statement here, but this isn't one.  Are you trying to express:

```
mods:recordInfo/mods:recordIdentifier
```

Reply:
So, this is the field list issue I was referring to in the initial PR. Having two colons together makes this into a field list. I think writing it as an Xpath without quotes makes it much more useable. I did want to confirm that we're fine with having field lists though.

Reply:
Not sure I follow exactly, but we probably should be doing:

.. code-block:: text

    mods:recordInfo/mods:recordIdentifier

Reply:
Ah, okay. We haven't previously used code blocks for our Xpaths, so I may go through and change this for the entire doc. That would solve the field list issue too.

Reply:
Yeah, again, this is just my idea.  To me, the xpath is code and would display better, but it definitely causes work.  @mathewjordan and @CanOfBees , thoughts on this?

Reply:
I'm partial to treating XPath as code, so I'd say use the code-blocks.

Reply:
surfing the internet, I just stumbled across syntax that might work for this:

1. `this is interpreted text`
2. :code:`this is interpreted text with a role`

The example I found wants to leverage highlighting, so we might not need to have `role` prefix. What about just having XPath in wrapped in backquotes (`)?

[stack overflow link](https://stackoverflow.com/questions/10870719/inline-code-highlighting-in-restructuredtext)

Reply:
This was resolved in https://github.com/UTKcataloging/mods_to_rdf/pull/27/commits/2bd47b57b297b1805e1b9a7e6f517ce517eda056